textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTester.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
```

Comment:
Can we get rid of this "RegexMatcherTester" class by adding a helper function to "RegexMatcherTest"?  Or at least we should rename "RegexMatcherTester" to something like "RegexMatcherTestHelper".